x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -94,7 +87,8 @@ export const buildAlertStatusesFilter = (statuses: Status[]): Filter[] => {
   ];
 };
 
-export const buildAlertsRuleIdFilter = (ruleId: string | null): Filter[] =>
+// TODO: Separate between rule.id and rule.rule_id
+export const buildAlertsRuleIdFilter = (ruleId: string | null, useRuleSOId = false): Filter[] =>
```

Comment:
Thoughts on separating to two specific filter functions, or just renaming `buildAlertsFilter` and specifying which ID? It gets quite confusing throughout the codebase when working with `rule.id` and `rule.rule_id`, so would like to start cleaning things up here and setting a clear pattern (and naming) between the two. Perhaps `rule.rule_id` is just referenced as `staticRuleId` in code going forward?

Reply:
Since updating the `useAddException` hook to support `ruleStaticId` removed the last usage of the `ruleId` filter I went ahead and just cleaned up this function to only use `ruleStaticId`. If we need a kuery filter for going against the `ruleId`/`uuid` we can create a new function -- didn't want to do that now as to add any confusion as to why you would use one vs the other (most UX should be using ruleStaticId when querying alerts going forward).